### PR TITLE
Morgan/generic converter component

### PR DESCRIPTION
**Pull request by MorganELLiot**
I think this is finally working right!

### CODE REVIEW

**Comment by komali2** [comment]
great commit titles

**Comment by komali2** [comment]
ah, ther'es some conflicts on main, i'll have to show you how to resolve that probably

File: src/components/Body.vue [hunk 1]
```diff
@@ -4,21 +4,51 @@
...
+        return Number((convert * 0.036).toFixed(2));
+      },
+      mphConvert(convert) {
+        return Number(Math.round(convert * 1.6 * 10) / 10);
+      },
+      kphConvert(convert) {
+        return Number(Math.round(convert * 0.62 * 10) / 10);
+      },
+      fConvert(convert) {
+        if(convert == 0) return Number(0);
```
Comment (komali2) [inline src/components/Body.vue]: change to `===` 

https://github.com/getify/You-Dont-Know-JS/blob/1st-ed/types%20%26%20grammar/ch4.md#loose-equals-vs-strict-equals

Comment (komali2) [reply src/components/Body.vue]: ah, um, btw, you might find that if you change to `===`, it stops working, you'll probably have to change to `if(Number(convert) === 0)`, read the above link to find out why

File: src/components/Body.vue [hunk 2]
```diff
@@ -4,21 +4,51 @@
...
+        else return Number(Math.round((convert - 32) * 5 / 9));
+      },
+      cConvert(convert) {
+        if(convert == 0) return Number(0);
```
Comment (komali2) [inline src/components/Body.vue]: same, change to `===`

File: src/components/Converter.vue [hunk 3]
```diff
@@ -0,0 +1,31 @@
+<template>
+  <div id="converter">
+    <p>{{ label }}</p>
+    <input v-model.number="convert" type="number" placeholder="Enter USD" />
```
Comment (komali2) [inline src/components/Converter.vue]: placeholder is hardcoded, should be a prop that's different depending on the converter

**Review by komali2 — CHANGES_REQUESTED** [review]

File: src/components/Body.vue (hunk 1 above)
Comment (MorganELLiot) [reply src/components/Body.vue]: It did want `if(Number(convert) === 0)` in order to work properly, cool.

**Comment by MorganELLiot** [comment]
I think the manual merge worked right, look ok?

**Pull request merged**